Makefile

```diff
@@ -1,23 +1,36 @@
 # SONiC make file
 
 NOJESSIE ?= 0
```

Comment:
Can we eliminate references to Jessie at this point?

Reply:
not yet. still keep jessie here in case people wants to develop jessie based docker.